src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const arrFromSource = sourceString
+    .trim()
+    .split(';')
+    .map((el) =>
+      el
+        .trim()
+        .split(':')
+        .map((item) => item.trim()),);
+
+  const obj = {};
```

Comment:
The variable name `obj` is too generic. Use a more descriptive name, such as `styleObject` or `cssObject`, to improve readability.